### ISSUE TITLE
Unknown javascript error in summary html plot

### ISSUE THREAD

**Issue by Caleb-Carr** [dms-vep/dms-vep-pipeline-3#68]
@WillHannon-MCB When running the pipeline, the summary plot data is correctly extracted and populates the csv file, but when looking at the html plots, the following error appears? I have searched the config file for any hidden chars that could be messing up the rendering but haven't found any

![image](https://github.com/dms-vep/dms-vep-pipeline-3/assets/43453129/8f5d7ffc-712a-4624-8638-476b326915a4)

summaries_config.yml
```
# Configuration for creating summary plots across assays.

# --------------------------------------------------------------------------------------
# Define how to create an integrated set of line plots and heat maps that summarizes
# antibody escape alongside other assays (eg, functional effects of mutations).
# --------------------------------------------------------------------------------------

summary:
  min_times_seen: 3  # only include mutations with times_seen >= this
  min_frac_models: 1  # only include mutations in >= this fraction of models
  alphabet: [A, C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, T, V, W, Y]  # amino acids to include
  init_floor_escape_at_zero: true  # initially floor site escape to be >= 0?
  init_site_escape_stat: sum  # initially show site escape as this statistic
  antibody_escape:  # show effects on antibody escape, empty dict if no antibodies being shown
    Arevirumab:  # name each set of antibodies
      stat: escape_median  # statistic to use for escape
      positive_color: "#56B4E9"  # light blue, positive heatmap color
      negative_color: "#E69F00"  # orange, negative heatmap color
      max_at_least: 1  # make heatmap max at least
      min_at_least: -1  # make heatmap min at least this
      antibody_list:  # antibodies to include: keys name used for `avg_escape`, values name for plot
        372D: 372D 
        121F: 121F 
        89F: 89F 
  func_effects: # effects of mutations on function
    293T entry:  # name of functional selection
      condition: 293T_entry  # condition specified in `avg_func_effects`
      effect_type: func_effects  # "func_effects" or "latent_effects"
      positive_color: "#009E73"  # green
      negative_color: "#F0E442"  # yellow
      max_at_least: 1
      min_at_least: 0
      init_min_value: -3  # initial min value in slider
  other_assays: {} # other assays that we show, empty dict if no assays being shown
  title: Effects of mutations to glycoprotein (GP) of Lassa virus Josiah strain on antibody escape and DG-mediated cell entry as measured in lentiviral deep mutational scanning
  legend: |
    This is is an interactive chart. Mouseover points for details and measurements, and
    use the top zoom bar to zoom in on specific sites. The options at the bottom let you
    only show escape for sites with some minimal functional effect, choose the site
    summary metric, or floor the escape at zero.

    The line plots at top show the escape at each site across all sera and for individual
    sera, and the heatmaps show the effects of individual mutations on sera escape or
    functions of the protein.
```

**Comment by WillHannon-MCB** [comment]
@Caleb-Carr, can you either post or DM me the offending HTML file that contains the chart specification?

**Comment by WillHannon-MCB** [comment]
Alright, I've spent some time looking at this now. I'm still not 100% sure what the exact cause of the issue is. I don't think I can figure out the issue unless I have some more information. 

I used this `Vega Spec` with the debugger to walkthrough the source code and see if I could pinpoint where the issue was occurring. It looks like it's an issue with how part of an "expression" is being parsed in the `Spec`. More specifically, a binary expression somewhere in the `Spec` – not very helpful since anything with a binary operator (`+`, `-`, `>=`, etc...) is an binary expression – has a value that being parsed as a number but isn't a valid number. This value seems to be `293T`. Essentially, it's parsing this expecting a number until it hits the `T` and throws an error. The only other thing I know is that the `293T` is part of the string `293T entry`  somewhere in the `Spec`. 

Unfortunately, I haven't been able to pinpoint where the problematic expression is in the `Spec`. As a shot in the dark,  I'd start by looking at this part of the `config.yml` to see if it's causing the issue:

```
  func_effects: # effects of mutations on function
    293T entry:  # name of functional selection <------------------------- This space could be an issue, maybe? 
      condition: 293T_entry  # condition specified in `avg_func_effects`
      effect_type: func_effects  # "func_effects" or "latent_effects"
      positive_color: "#009E73"  # green
      negative_color: "#F0E442"  # yellow
      max_at_least: 1
      min_at_least: 0
```

I might come back to this later and see if I can figure it out.
